plugins/playlist/src/components/PlaylistEditDialog/PlaylistEditDialog.tsx

```diff
@@ -74,13 +75,39 @@ export const PlaylistEditDialog = ({
 }: PlaylistEditDialogProps) => {
   const classes = useStyles();
   const identityApi = useApi(identityApiRef);
+  const playlistApi = useApi(playlistApiRef);
+  const [playlistArray, setPlaylistArray] = useState<PlaylistMetadata[]>([]);
+  const [editingOtherFields, setEditingOtherFields] = useState(false);
+
+  useEffect(() => {
+    const fetchPlaylists = async () => {
+      const playlists = await playlistApi.getAllPlaylists({ editable: true });
+      setPlaylistArray(playlists);
+    };
+
+    fetchPlaylists();
+  }, [playlistApi]);
```

Comment:
This still just does it once. Oh well the dialog is short lived so it's probably fine.